prospector/processing/natural_language_processing.py

```diff
@@ -0,0 +1,91 @@
+import re
```

Comment:
I would call this module nlp.py for conciseness (chances that one would not get the acronym, in this context, are quite low)